backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/RawDataFilterBuilder.kt

```diff
@@ -0,0 +1,37 @@
+package app.ehrenamtskarte.backend.stores.importer
+
+import app.ehrenamtskarte.backend.stores.importer.types.LbeAcceptingStore
+
+data class FilterBuilder(val store: LbeAcceptingStore, val monitor: ImportMonitor) {
+
+    fun filterLongitudeAndLatitude()
+            = if (store.longitude.isNullOrBlank() || store.latitude.isNullOrBlank()) {
+        monitor.addMessage("$store was filtered out because longitude or latitude were invalid")
+        false
+    } else {
+        true
+    }
+
+    fun filterPostalCode() = if(store.postalCode == null || store.postalCode?.trim()?.length == 5) {
```

Comment:
this one might be a bit too harsh. I think there are some instances where the accepting store looks fine but the postal code included e.g. the city name. Maybe a warning would be better here instead of blocking it completely

Reply:
Problem: What to with the data else?
I don not like to extend the postal code field to an text field of arbitrary length just because of some instances that do not fit into.